src/app-bar.jsx

```diff
@@ -127,6 +159,7 @@ const AppBar = React.createClass({
       showMenuIconButton: true,
       title: '',
       zDepth: 1,
+      position: 'fixed',
```

Comment:
IMHO, main use case here is 'fixed', because, before, AppCanvas [overrided it's position to fixed](https://github.com/callemall/material-ui/pull/1321/files#diff-9f235eb5b7c83ffc1cae3847947357afL59). So, putting it to 'static' would be more of a breaking change. But, of course it depends of how people are using it.
I can change it to 'static' if you still say so.